hoprd/rest-api/src/messages.rs

```diff
@@ -9,11 +9,10 @@ use axum::{
 };
 use futures::{sink::SinkExt, stream::StreamExt};
 use futures_concurrency::stream::Merge;
-use libp2p_identity::PeerId;
 use serde::Deserialize;
 use serde_json::json;
 use serde_with::{serde_as, Bytes, DisplayFromStr, DurationMilliSeconds};
-use std::{sync::Arc, time::Duration};
+use std::{f32::consts::E, sync::Arc, time::Duration};
```

Comment:
Huh?

Reply:
Huh ? Removing. I don't know how it ended up here.